lib/src/common/variables.js

```diff
@@ -680,6 +683,25 @@ export const componentTokens = {
     activeUnderlineColor: globalTokens.hal_black,
     focusColor: globalTokens.hal_blue_l_50,
   },
+  navTabs: {
+    selectedBackgroundColor: globalTokens.white,
+    unselectedBackgroundColor: globalTokens.white,
+    hoverBackgroundColor: globalTokens.color_grey_a_100,
+    pressedBackgroundColor: globalTokens.color_grey_a_200,
+    selectedFontColor: globalTokens.color_grey_700,
+    unselectedFontColor: globalTokens.color_grey_700,
+    disabledFontColor: globalTokens.hal_grey_l_60,
+    focusOutline: globalTokens.hal_blue_l_50,
+    selectedUnderlineColor: globalTokens.hal_purple_s_38,
+    dividerColor: globalTokens.hal_grey_l_75,
+    fontFamily: globalTokens.type_sans,
+    fontSize: globalTokens.type_scale_03,
+    fontStyle: globalTokens.type_normal,
+    fontWeight: globalTokens.type_regular,
+    selectedIconColor: globalTokens.color_grey_700,
+    unselectedIconColor: globalTokens.color_grey_700,
+    disabledIconColor: globalTokens.hal_grey_l_60
```

Comment:
`selectedIconColor` and `unselectedIconColor` have same value (`#666666`). They should have different value I think.